pkg/pipeline/PipelineBuilder.go

```diff
@@ -4108,7 +4161,7 @@ func (impl PipelineBuilderImpl) BulkDeleteCdPipelines(impactedPipelines []*pipel
 			EnvironmentName: pipeline.Environment.Name,
 		}
 		if !dryRun {
-			deleteResponse, err := impl.DeleteCdPipeline(pipeline, ctx, deleteAction, true, userId)
+			deleteResponse, err := impl.DeleteCdPipeline(pipeline, ctx, deleteAction, true, userId, nil)
```

Comment:
null is not handled inside ``DeleteCdPipeline`` func

Reply:
removed the parameter itself so no need for nil handling now